bin/sandbox

```diff
@@ -109,7 +124,12 @@ unbundled bundle exec rails generate solidus:install \
   $@
 
 unbundled bundle exec rails generate solidus:auth:install --auto-run-migrations
-unbundled bundle exec rails g solidus_starter_frontend:install
+
+if [[ -z $GENERATE_FRONTEND ]]; then
+  unbundled bundle exec rails g solidus_starter_frontend:install
+else
+  unbundled ../exe/${extension_name} --auto-accept
```

Comment:
Even if we're running the executable from our local copy, it'll run load `lib` files from the global `solidus_starter_frontend` gem, which could be the one released in rubygems.org.

However, we can execute ruby with the `-I` option to add a directory to the `$LOAD_PATH`. In addition, I don't think we need `unbundled` here, as we're not running `bundle`. So, I think that we could remove the checking about `solidus_starter_frontend` being installed and instead change this line to:

```suggestion
  ruby -I../lib ../exe/${extension_name} --auto-accept
```

Reply:
Thanks @waiting-for-dev ! I updated the `solidus_starter_frontend` call here and remove the error checking at https://github.com/nebulab/solidus_starter_frontend/pull/171#discussion_r678933726.

Reply:
Thanks, @gsmendoza! Awesome! :heart: